workalendar/asia.py

```diff
@@ -60,6 +60,124 @@ def get_variable_days(self, year):
         return days
 
 
+class Malaysia(LunarCalendar, WesternCalendar, IslamicMixin):
+    "Malaysia"
+    include_nuzul_al_quran = True
+    include_eid_al_fitr = True
+    length_eid_al_fitr = 2
+    eid_al_fitr_label = "Hari Raya Puasa"
+    include_day_of_sacrifice = True
+    day_of_sacrifice_label = "Hari Raya Haji"
+    include_islamic_new_year = True
+    include_prophet_birthday = True
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (2, 1, "Federal Territory Day"),
+        (5, 1, "Workers' Day"),
+        (8, 31, "National Day"),
+        (9, 16, "Malaysia Day"),
+        (12, 25, "Christmas Day"),
+    )
+
+    MSIA_DEEPAVALI = {
+        2010: date(2010, 11, 5),
+        2011: date(2011, 10, 26),
+        2012: date(2012, 11, 13),
+        2013: date(2013, 11, 2),
+        2014: date(2014, 10, 22),
+        2015: date(2015, 11, 10),
+        2016: date(2016, 10, 29),
+        2017: date(2017, 10, 18),
+        2018: date(2018, 11, 7),   # This might change
+        2019: date(2019, 10, 27),  # This might change
+        2020: date(2020, 11, 14),  # This might change
+    }
+
+    MSIA_THAIPUSAM = {
+        2010: date(2010, 1, 30),
+        2011: date(2011, 1, 20),
+        2012: date(2012, 2, 7),
+        2013: date(2013, 1, 28),
+        2014: date(2014, 1, 17),
+        2015: date(2015, 2, 3),
+        2016: date(2016, 1, 25),
+        2017: date(2017, 2, 9),
+        2018: date(2018, 1, 31),   # This might change
+    }
+
+    # Following code is largely borrowed from Singapore class.
```

Comment:
fair enough.
When you PR will be accepted, I'll work on a refactor that will get this code into a common module, for more reusability.
But don't worry about this here, it can wait.